core/client/app/controllers/setup/three.js

```diff
@@ -2,40 +2,130 @@ import Ember from 'ember';
 import ValidationEngine from 'ghost/mixins/validation-engine';
 
 var SetupThreeController = Ember.Controller.extend(ValidationEngine, {
```

Comment:
As far as I can tell this controller doesn't need to use `ValidationEngine`, so it shouldn't need to be imported or mixed-in here.